.github/workflows/test-pr.yml

```diff
@@ -30,6 +30,30 @@ jobs:
             git push origin HEAD:${GITHUB_HEAD_REF}
           fi
 
+  arch:
+    name: 'Arch Linux Package'
+    runs-on: [self-hosted, linux, normal]
+    steps:
+      - uses: actions/checkout@v3
+      - name: 'Build and Test Package'
+        uses: ./.github/actions/test-package
+        with:
+          dockerfile: package/arch/Dockerfile
+          os: archlinux
+          distro: base
+          build-package: package/arch/build-package
+          test-package: package/arch/test-package
+          pkg-name: kframework_arch_x86_64.pkg.tar.zst
```

Comment:
Here we tell it where the package should be moved to when calling the `test-package` action (where `BUILD_PACKAGE` should move it to, and `TEST_PACKAGE` should assume it's at).